apply-labels/.env.example

```diff
@@ -0,0 +1,8 @@
+BASE_URL=https://app.datasaur.ai
```

Comment:
test... ignore this

Reply:
soal env keknya nanti perlu diseragamin aja ya, jadi semua folder pakenya env gitu, krn sekarang yg lain kan masih pake param

Reply:
nah iya - awalnya aku notice di requirements.txt itu ada python-dotenv
tapi kok ga dipake...

jadi kemarin sekalian coba

command ini sekarnag jg bisa kalau mau provide manual, tapi defaultnya akses .env jg